assets/js/googlesitekit/notifications/components/Notification/index.js

```diff
@@ -81,6 +81,7 @@ Notification.propTypes = {
 	id: PropTypes.string,
 	className: PropTypes.string,
 	gaTrackingEventArgs: PropTypes.shape( {
+		category: PropTypes.string,
 		label: PropTypes.string,
 		value: PropTypes.string,
```

Comment:
This seems to be okay for now, so long as these are passed as positional args to `trackEvent` but I'd like to move to a more modern application where we call it as a wrapper for `gtag` such as `trackEvent(eventName, args) => gtag('event', eventName, args)`. The difference being that in that paradigm, `category` would be `event_category` (the name of the dimension`, same with `label`. See https://github.com/google/site-kit-wp/blob/37b72fa3a5217570defbb26b5e85520c9ede4ca8/assets/js/util/tracking/createTrackEvent.js#L49-L51

This is something I've been meaning to open an issue to do.

Reply:
This definitely would make things more obvious.